webapp/views/index.erb

```diff
@@ -0,0 +1,46 @@
+<html>
+<head>
+<title>Continue benchmarking microsite</title>
+<link rel="stylesheet" type="text/css" href="css/style.css">
+<script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+<script src="js/Chart.js"></script>
+</head>
+<body>
+<h1>Continious benchmarking microsite</h1>
+
+<p>
+Taleggio airedale mozzarella. Cheesy feet mascarpone mozzarella ricotta stilton paneer fondue cauliflower cheese. Cheese triangles cheesecake fondue caerphilly chalk and cheese cheddar cream cheese cut the cheese. Pepper jack boursin halloumi camembert de normandie roquefort blue castello paneer goat. Halloumi dolcelatte babybel stinking bishop.
+</p>
+<p>
+Queso fromage halloumi. Port-salut melted cheese blue castello melted cheese pecorino everyone loves roquefort port-salut. Melted cheese when the cheese comes out everybody's happy swiss cheese on toast feta cut the cheese feta stilton. Fromage bavarian bergkase blue castello babybel goat bavarian bergkase when the cheese comes out everybody's happy.
+</p>
+
+<div id="wrapper">
+  <div id="body">
+    <div class="header">
+      <h3>Events</h3>
```

Comment:
It would be nice to have a link to the code used to generate these tests. maybe a link to github?